### ISSUE TITLE
[Feature] Local commenting feature

### ISSUE THREAD

**Issue by HTSamurai** [TeamNewPipe/NewPipe#5147]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want
Basically a textbox in the comment section that you can type acomment, and you cab have a name and maybe profile pic for it to show, and have it show at the top of the comments and maybe in video like on soundcloud. Just so you can make notes probably. Having it pop up in video would be optional

#### Is your feature request related to a problem? Please describe it
I don't care if people see it, but I like to make comments to see when I watch.

#### Additional context
*Mock up images to show what I mean. All comments wouldn't show up for other NewPipe or YouTube users, similar to subscriptions*
![IMG_20201211_122242](https://user-images.githubusercontent.com/71007160/101903376-15239800-3bac-11eb-881c-75f376b82203.jpg)
![IMG_20201211_123059](https://user-images.githubusercontent.com/71007160/101903899-e8bc4b80-3bac-11eb-906d-299b39637419.jpg)
*2nd mock up shows the soundcloud style popup*

#### How will you/everyone benefit from this feature?
People will be able to take notes in videos easier! It would be better than using a note app because it shows up in the video.

**Comment by MD77MD** [comment]
I wish if this could be implemented.. what a great idea.

**Comment by HTSamurai** [comment]
i asked on IRC and @mhmdanas said it is a good idea, so i think it might be added 😀

**Comment by mhmdanas** [comment]
Well, I don't actually decide what features are added.

**Comment by HTSamurai** [comment]
Very good point.

**Comment by HTSamurai** [comment]
Nothing's happened here in ages! Should I close the issue I doubt it is being added at this point after so long

**Comment by mhmdanas** [comment]
No, it doesn't have to be closed. Also, four months can hardly be called "ages."

**Comment by HTSamurai** [comment]
Feels like it to me

**Comment by HTSamurai** [comment]
Ima make new mockups

**Comment by HTSamurai** [comment]
And the whole issue

**Comment by HTSamurai** [comment]
There we go this is much better

**Comment by HTSamurai** [comment]
I updated the issue to fit newer versions of newpipe better.

**Comment by SameenAhnaf** [comment]
@litetex Close in favor of https://github.com/TeamNewPipe/NewPipe/issues/8322

**Comment by litetex** [comment]
@SameenAhnaf This issue is more specific about local comments, the other was not ;)
Closed the other one instead.
Anyway thank you for the hint.
